extensions/data-workspace/src/common/workspaceTreeDataProvider.ts

```diff
@@ -24,6 +25,7 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
 	readonly onDidChangeTreeData?: vscode.Event<void | WorkspaceTreeItem | null | undefined> | undefined = this._onDidChangeTreeData?.event;
 
 	async refresh(): Promise<void> {
+		Logger.log(`Refreshing projects tree`);
```

Comment:
I didn't add the duration for refresh because the duration of `getProjectsInWorkspace()` already gets measured in that function and also because the time taken of what firing `this._onDidChangeTreeData` calls wouldn't get included in the calculated "refresh" time.